app/dashboard/models.py

```diff
@@ -2365,6 +2365,17 @@ def is_staff(self):
         """
         return self.user.is_staff if self.user else False
 
+    @property
+    def success_rate(self):
+        network = self.get_network()
+        num_completed_bounties = self.bounties.filter(
+            idx_status__in=['done'], network=network).count()
+        terminal_state_bounties = self.bounties.filter(
+            idx_status__in=Bounty.TERMINAL_STATUSES, network=network).count()
```

Comment:
don't know how to solve the cooperative bounty case, since I can be paid out but the bounty still open.

Reply:
Is kind of the same situation on the dashboard, should we fetch fulfillments also?